api/src/test/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceTest.java

```diff
@@ -41,23 +40,26 @@
 public class FeaturedWorkspaceTest {
 
   @MockBean private FeaturedWorkspaceDao mockFeaturedWorkspaceDao;
-  @MockBean private FeaturedWorkspaceMapper mockFeaturedWorkspaceMapper;
```

Comment:
mock mappers are difficult to work with.  replaced these with the real versions.